src/index.tsx

```diff
@@ -5,13 +5,13 @@ const contextEnabled = Object.prototype.hasOwnProperty.call(React, "createContex
 const hooksEnabled = Object.prototype.hasOwnProperty.call(React, "useMemo") && Object.prototype.hasOwnProperty.call(React, "useCallback");
 
 export interface IPrintContextProps {
-    handlePrint: () => void,
+    handlePrint: (event: React.MouseEvent, lazyOption?: { content: (() => React.ReactInstance) }) => void,
```

Comment:
Any thoughts on just passing the content as the arg rather than using the object?
```suggestion
    handlePrint: (event: React.MouseEvent, content?: () => React.ReactInstance) => void,
```

Reply:
That's okay, will do so.